Asterix/tests/param_file_tests.ini

```diff
@@ -184,9 +185,9 @@ onbench=True
     # lyot diameter (in meters). 
     # Only use in the case of a RoundPup Lyot stop.
     # not use for propagation. 
-    # Current value is 8.035mm = 8.1*0.97 
-    # (rayon Lyot * de-zoom entrance pupil plane / Lyopt plane)
-    diam_lyot_in_m  = 8.035e-3
+    # Current value is 7.984mm = 7.9*1.0107
```

Comment:
Same here, please make sure the number in the comment is the same like the number that is actually used.

Reply:
done